Dockerfile.ci-rp

```diff
@@ -25,24 +55,26 @@ COPY cmd cmd
 COPY pkg pkg
 COPY test test
 
-# lint
-RUN golangci-lint run --verbose
+# Ensure JS assets are available before generating Go code
+COPY --from=portal-build /build/portal/v2/${BUILD_PATH} /app/pkg/portal/assets/v2/build
```

Comment:
bug: ${BUILD_PATH} is not definined at this point of the multi-stage build. I think we should just hard-code the path.

Reply:
Fixed the bug by hardcoding the path in the COPY command to ensure the JS assets are available before generating the Go code.